ci/buildwheel.sh

```
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+function setup_root() {
+  mkdir /root/.pip && ln -s /workspace/pip.conf /root/.pip/pip.conf
+  ln -s /workspace/netrc                 /root/.netrc
+  ln -s /workspace/cloudbuild_pypirc     /root/.pypirc
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Delete it.

(SC2112)